lib/active_admin/resource/attributes.rb

```diff
@@ -0,0 +1,44 @@
+module ActiveAdmin
+
+  class Resource
+    module Attributes
+
+      def default_attributes
+        resource_class.columns.each_with_object({}) do |c, attrs|
+          unless reject_col?(c)
```

Comment:
Oh, I didn't know rails had such method, nice. Where are we using it? In `CSVBuilder`, you seem to be using Rails `columns` method, not content columns.